opm/parser/eclipse/EclipseState/InitConfig/InitConfig.cpp

```diff
@@ -0,0 +1,67 @@
+/*
+  Copyright 2015 Statoil ASA.
+
+  This file is part of the Open Porous Media project (OPM).
+
+  OPM is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  OPM is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with OPM.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <opm/parser/eclipse/EclipseState/InitConfig/InitConfig.hpp>
+#include <opm/parser/eclipse/Deck/Deck.hpp>
+
+#include <iostream>
+
+namespace Opm {
+
+    InitConfig::InitConfig(DeckConstPtr deck) {
+        m_restartInitiated = false;
+        m_restartStep = 0;
+        m_restartRootName = "";
+
+        initRestartKW(deck);
+    }
+
+    void InitConfig::initRestartKW(DeckConstPtr deck) {
+        if (deck->hasKeyword("RESTART") && deck->hasKeyword("SKIPREST")){
+            DeckKeywordConstPtr restart_kw = deck->getKeyword("RESTART");
+            DeckRecordConstPtr restart_dataRecord = restart_kw->getRecord(0);
+            DeckItemPtr restart_rootname_item = restart_dataRecord->getItem(0);
+            const std::string restart_rootname_string = restart_rootname_item->getString(0);
```

Comment:
If *only* one of `RESTART` or `SKIPREST` is found we should throw. We *really* should get the `OpmLog` system in proper use - then it will be natural to issue a warning there.